tests/karma.conf.js

```diff
@@ -14,15 +16,20 @@ if (process.env.TEST_FILE) {
     }
   });
 } else {
-  FILES.push('tests/**/*.test.js');
+  glob.sync('tests/**/*.test.js').forEach(function (filename) {
+    FILES.push(filename);
+  });
```

Comment:
I didn't quite understand why the previous code didn't work anymore, it gave me 16 tests instead of 355 tests. Doing the glob myself like it's done when you use `TEST_FILE=atestfile npm run test:chrome` (the if above) fixed it.